bedrock/mozorg/templates/mozorg/contribute/contribute-tasks-survey.html

```diff
@@ -0,0 +1,93 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% block page_title %}{{_('Thank you!')}}{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('contribute-signup-optimizely') and LANG == 'en-US' %}
```

Comment:
If we put the `LANG == 'en-US'` check first, then that will eliminate the memcache and/or db hit for all non-US traffic.